### PR TITLE
first draft index

### PR DESCRIPTION
**Pull request by sophiecs333**
This is not done at all.
I pushed it because I wanted to check your pull requests.
I'll keep working on it later but you can already review.

### CODE REVIEW

File: app/models/scan.rb [hunk 1]
```diff
@@ -3,4 +3,7 @@ class Scan < ApplicationRecord
 
   validates :location, presence: true
   validates_inclusion_of :sticker_destroyed, in: [true, false]
+
+  geocoded_by :location
+  after_validation :geocode, if: :will_save_change_to_location?
```
Comment (dumboluzz) [inline app/models/scan.rb:8]: try to use reverse_gecoded_by

**Review by dumboluzz — CHANGES_REQUESTED** [review]
add seeds to lat and long

File: app/models/scan.rb (hunk 1 above)
Comment (sophiecs333) [reply app/models/scan.rb]: Hello, I made the updates can you please merge? :D

**Review by dumboluzz — APPROVED** [review]
👍🏻

**Pull request merged**